docs/semgrep-app/pricing-and-billing.mdx

```diff
@@ -1,53 +1,163 @@
 ---
 slug: pricing-and-billing
-description: >-
-  Pricing and billing for Semgrep App
+append_help_link: true
+title: Pricing and billing
+description: "Semgrep CLI and CI are free to use. Semgrep App has both free and paid tiers, each with their own features and levels of support."
 ---
 
 import MoreHelp from "/src/components/MoreHelp"
 
+
 # Pricing and billing
 
-## Can I use Semgrep for free?
+The following Semgrep tools are free to use:
+
+* Semgrep CLI
+* Semgrep CI
+* Semgrep App (through the Community tier)
+
+Semgrep CLI is free open-source software (FOSS), licensed under LGPL 2.1. Semgrep CI is source-available. All features for both Semgrep CLI and Semgrep CI are free.
+
+Semgrep App, which builds on Semgrep CLI and CI, enables developer teams to create their own institutional code standards, enforce them, and analyze findings. Semgrep App has both free and paid tiers.
```

Comment:
Maybe we can shortly say again what is Semgrep CLI, what is Semgrep CI... I know CLI and CI are common terms, but it might be good to mention this again and again.

Reply:
Added a description for both!